lib/ledger_sync/resource.rb

```diff
@@ -64,17 +74,27 @@ def self.klass_from_resource_type(obj)
       LedgerSync.const_get(LedgerSync::Util::StringHelpers.camelcase(obj))
     end
 
-    def self.reference(name, type)
+    def self.has_one(name, type)
```

Comment:
To my previous comment of using `belongs_to`, I need to think about this, but do we need to draw a distinction between `belongs_to` and `has_one` for this lib?  @mattgmarcus may have insights here too.

Reply:
We don't do two way relationships yet. Thats why I used `has_one` instead of `belongs_to`

Reply:
By two way relationship I mean we dont take account resource and as for what resources is that account used. Right now we do only reference and `has_many` matches better with `has_one` in this case.

Reply:
Isn't that just a "we don't yet?"  Because we know if they are `has_one` or `belongs_to` intuitively right now, but there can be some objects that self-reference.